src/views/KidVids/index.jsx

```
@@ -0,0 +1,244 @@
+import React, { Fragment, useCallback, useEffect, useState } from "react";
+import BreadCrumb from "../../components/breadcrumb";
+import { FaPlayCircle, FaPlus } from "react-icons/fa";
+import Header from "../../components/header";
+import { handleSearch } from "../../utils/common/fetchDataHelpers";
+import { useDispatch, useSelector } from "react-redux";
+import ThumbnailCard from "../../components/card/ThumbnailCard";
+import { Spinner } from "../../components/theme/Loader";
+import { nameElipse } from "../../utils/common/nameElipse";
+import Modal from "../../components/modal/Modal";
+import { AddFanStar, FanStarPlayer } from "../../services/fanstarzone";
```

ESLint

[error] 'AddFanStar' is defined but never used.

(no-unused-vars)